tests/scripting_api/test_engine.py

```diff
@@ -135,9 +135,78 @@ def test_engine_create_folder():
 
 def test_engine_create_folder_subfolder():
     engine, folder = create_engine()
-    # Temporary: Don't put folder on disk.
+    # Temporary: prevent persisting (which fails b/c folder doesn't exist).
     test_folder = engine.create_folder("New folder",
             parent_folder=folder, temporary=True)
     assert_that(engine.configManager.allFolders, not_(has_item(test_folder)), "creates top-level folder instead of subfolder")
     assert_that(folder, is_(equal_to(test_folder.parent)),
-    "Doesn't add parent folder as parent of subfolder")
+        "Doesn't add parent folder as parent of subfolder")
+    assert_that(folder.folders,
+        has_item(test_folder),
+            "Doesn't create subfolder in correct folder")
+
+
+# These tests shouldn't really be here since they're for configmanager
+# class, not engine. But I'm not confident enough with settings up
+# tests to move them.
```

Comment:
Tests are autodetected by pytest based on the file name, which must be `test_whatever.py` or `whatever_test.py`.
Just create a new python file called `test_configmanager.py` or create a whole package named `configmanager` under `tests` with a `test_<some_name>.py` inside. I’ll leave that choice to you, as you see fit.

Move the tests to the new test module, import things from test libraries and autokey as needed and copy any setup boilerplate code you currently use there to decouple them. After you decoupled them, you can alter the copy to suit the specific needs.
Any non-test functions and structures I defined at the beginning of test files are written to reduce test code length and write concise code, which is tailored to the specific needs of the test module. In my opinion, those should not be shared using imports as that introduces dependencies.